lib/main.dart

```diff
@@ -93,7 +93,9 @@ class MyApp extends StatelessWidget {
         shiftInstance.notify();
         shiftInstance.id = await dao.insertShift(shiftInstance);
       }
-      return true;
+      logger.d("User is logged -> ${userModel.logged}");
+      logger.d("ShiftScheduler have following payload -> $shiftInstance");
+      return userModel.logged;
```

Comment:
```suggestion
      return userModel.logged && GetIt.isRegistered<AuthProvider>();
```

If the user will upgrade the version we don't know which version is had used. In this way, we can check also this part. It is logged && is the provider is registered